README.md

```diff
@@ -71,13 +85,17 @@ Components in beta stage may include breaking changes, new features, and bug fix
 
 <!-- AUTO-GENERATED-CONTENT:END *-->
 
-### Deprecated
+<h3 id="deprecated">
+  Deprecated
+</h3>
 
 | Terra Package      | Version | Status | Dependencies |
 |--------------------|---------|--------|--------------|
 | [terra-app-delegate](https://github.com/cerner/terra-framework/tree/master/packages/terra-app-delegate) | [![NPM version](https://badgen.net/npm/v/terra-app-delegate)](https://www.npmjs.org/package/terra-app-delegate) | ![Deprecated](https://badgen.net/badge/status/Stable/grey) | [![terra-app-delegate]](https://badgen.net/badge/dependencies/n%2Fa/grey) |
```

Comment:
Unrelated to your changes, but we have an extra `]` in [![terra-app-delegate]](https://badgen.net/badge/dependencies/n%2Fa/grey) which causes the link to render wrong